frameworks/PHP/symfony/composer.json

```diff
@@ -1,49 +1,69 @@
 {
-	"type" : "project",
-	"license" : "proprietary",
-	"require" : {
-		"php" : "^7.1.3",
-		"doctrine/doctrine-bundle" : "^1.6",
-		"doctrine/orm" : "^2.5",
-		"symfony/console" : "^4.0",
-		"symfony/flex" : "^1.0",
-		"symfony/framework-bundle" : "^4.0",
-		"symfony/twig-bundle" : "^4.0",
-		"symfony/yaml" : "^4.0",
-		"twig/twig" : "^2.4"
-	},
-	"require-dev" : {
-		"symfony/dotenv" : "^4.0"
-	},
-	"config" : {
-		"preferred-install" : {
-			"*" : "dist"
-		},
-		"sort-packages" : true
-	},
-	"autoload" : {
-		"psr-4" : {
-			"App\\" : "src/"
-		}
-	},
-	"autoload-dev" : {
-		"psr-4" : {
-			"App\\Tests\\" : "tests/"
-		}
-	},
-	"conflict" : {
-		"symfony/symfony" : "*"
-	},
-	"extra" : {
-		"symfony" : {
-			"id" : "01C0MBGZ184NXGJSE6HFTV8B43",
-			"allow-contrib" : false
-		}
-	},
-	"scripts": {
-		"auto-scripts": {
-			"cache:clear": "symfony-cmd",
-			"assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
-		}
-	}
+    "type": "project",
+    "license": "proprietary",
+    "require": {
+        "php": "^7.3.7",
+        "ext-ctype": "*",
+        "ext-iconv": "*",
+        "ext-mbstring": "*",
+        "symfony/console": "4.3.*",
+        "symfony/dotenv": "4.3.*",
+        "symfony/flex": "^1.3.1",
+        "symfony/framework-bundle": "4.3.*",
+        "symfony/orm-pack": "^1.0",
```

Comment:
you don't need to require the orm-pack. you should require the real packages, so you don't need to replace the doctrine-migrations-bundle at all.